src/Schema/Events/Search.ts

```diff
@@ -71,7 +71,7 @@ export interface SelectedItemFromSearch {
  *  ```
  *  {
  *    action: "searchedWithNoResults",
- *    context_module: "header",
+ *    context_module: "topTab",
```

Comment:
Good reminder that the more generic `header` ContextModule will be less often used in favor of these more specific pill/tab values.

cc/ @abhitip